### DISCUSSION TITLE
how to open new file when copy or create

### DISCUSSION THREAD

**Discussion by tkkcc** [nvim-neo-tree/neo-tree.nvim#1385] [Q&A] [answered]
currently i need to search the new filename and enter

**Comment by pysan3** [comment]
No easy way. However the cursor should move to focus the created file. Isn't that your case?

**Comment by pysan3** [comment, marked as answer]
@tkkcc Do you mean to open a buffer for the created files?

That is easy. Read `:h neo-tree-events` and you should find _file_added_ event.

**Comment by tkkcc** [comment]
Thanks! The file_added event handler is enough for me. 

And I just found neo-tree automatically focus new file when copy and create, and i can simply open it. It's strange i didn't notice this before. (°ー°〃)
